src/routes/passage/[passageId]/+page.svelte

```diff
@@ -81,6 +81,8 @@
         multiClipAudioStates = {};
         baseFetchPromise = (async () => {
             passage = await fetchPassage($page.params.passageId!);
+            cbbterSelectedStepNumber = 1;
```

Comment:
Does this need to account for if the step is available? Like some of the resources in Russian I believe don't have step 1?

Reply:
@carlosmorales125 just wanted to make sure this came through ^

Reply:
Oh gotcha, seems like a separate bug then. I'll approve.